modules/renter/contractor/downloader.go

```diff
@@ -72,34 +71,19 @@ func (hd *hostDownloader) Sector(root crypto.Hash) ([]byte, error) {
 	if hd.invalid {
 		return nil, errInvalidDownloader
 	}
-	oldSpending := hd.downloader.DownloadSpending
-	start := time.Now()
 	contract, sector, err := hd.downloader.Sector(root)
-	duration := time.Since(start)
 	if err != nil {
 		return nil, err
 	}
-	delta := hd.downloader.DownloadSpending.Sub(oldSpending)
-
-	hd.speed = uint64(duration.Seconds()) / modules.SectorSize
 
 	hd.contractor.mu.Lock()
-	hd.contractor.financialMetrics.DownloadSpending = hd.contractor.financialMetrics.DownloadSpending.Add(delta)
 	hd.contractor.contracts[contract.ID] = contract
 	hd.contractor.saveSync()
 	hd.contractor.mu.Unlock()
 
 	return sector, nil
 }
 
-// Speed returns the most recent download speed of this host, in bytes per
-// second.
-func (hd *hostDownloader) Speed() uint64 {
-	hd.mu.Lock()
-	defer hd.mu.Unlock()
-	return hd.speed
-}
```

Comment:
yeah I removed it because it wasn't used anywhere. The code is simple enough that to restore afterwards. It's also wrong: it calculates speed using `time / sectorsize` instead of the other way around.